index.html

```diff
@@ -0,0 +1,231 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta charset="utf-8">
+		<link rel="stylesheet" type="text/css" href="https://cdn.mxpnl.com/libs/mixpanel-platform/css/reset.css">
+		<link rel="stylesheet" type="text/css" href="https://cdn.mxpnl.com/libs/mixpanel-platform/build/mixpanel-platform.v0.latest.min.css">
+		<link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/chosen/1.1.0/chosen.min.css">
+		<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.10.11/css/jquery.dataTables.css">
+		<!-- smoothness library improves the appearance of the datepicker -->
+		<link rel="stylesheet" href="https://code.jquery.com/ui/1.11.4/themes/smoothness/jquery-ui.css">
+		<script src="https://cdn.mxpnl.com/libs/mixpanel-platform/build/mixpanel-platform.v0.latest.min.js"></script>
+		<script src="https://cdnjs.cloudflare.com/ajax/libs/chosen/1.1.0/chosen.jquery.min.js"></script>
+		<script type="text/javascript" charset="utf8" src="https://cdn.datatables.net/1.10.11/js/jquery.dataTables.js"></script>
+		<script src="customQueryGenerator.js"></script>
+		<script src="exportToCSV.js"></script>
+		<script src="generalFunctionsAndVariables.js"></script>
+		<script src="graphFunctions.js"></script>
+	</head>
+	<body class="mixpanel-platform-body">
+		<div class="mixpanel-platform-section">
+			<!-- Event Select -->
+			<div id="labelSelectEvent" class="mixpanel-platform-label" 
+				style="width: 258px; padding: 10 px">Select Event</div>
+			<div id="eventSelect" style="float: left;"></div>
+			
+			<!-- Select Operating System -->
+			<div style="clear: both;"></div>
+			<div id="labelSelectPlatforms" class="mixpanel-platform-label" 
+				style="width: 258px; padding: 10 px">Select Platform(s)</div>
+			<select data-placeholder="Select Platform(s), leave empty for all" 
+				class="chosen-platform" multiple="true" style="width: calc(100% - 300px); float:right" ></select>
+			
+			<!-- Select App Name(s) -->
+			<div style="clear: both;"></div>
+			<div id="labelSelectAppNames" class="mixpanel-platform-label" style=
+				"width: 258px; padding: 10 px">Select Application(s)</div>
+			<select data-placeholder="Select App(s), leave empty for all" class=
+				"chosen-appName" multiple="true" style="width: calc(100% - 300px); float:right" ></select>
+			<!-- Select Device Models -->

```

Comment:
I'd be consistent with your spacing...  Looks like you add a newline before each of these comments above.  You should probably continue to do so.